### ISSUE TITLE
panic: Segmentation fault at address 0x11

### ISSUE THREAD

**Issue by ghyath5** [oven-sh/bun#11702]
### How can we reproduce the crash?

working on Nuxt project with nitro server running with bun

### Relevant log output

```shell
Bun v1.1.12 (43f0913c) macOS Silicon
Args: "node" "/Users/ghiathdarwish/work/cobusinesses-bun/node_modules/.bin/nuxt" "dev"
Features: jsc Bun.stdin(4) Bun.stdout(2) abort_signal dotenv(3) fetch(114) http_server(2) spawn transpiler_cache(60) tsconfig_paths(14) tsconfig(42) 
Builtins: "bun:jsc" "bun:main" "detect-libc" "node:assert" "node:async_hooks" "node:buffer" "node:child_process" "node:constants" "node:crypto" "node:dns" "node:events" "node:fs" "node:fs/promises" "node:http" "node:https" "node:module" "node:net" "node:os" "node:path" "node:perf_hooks" "node:process" "node:querystring" "node:readline" "node:stream" "node:stream/promises" "node:string_decoder" "node:tls" "node:tty" "node:url" "node:util" "node:util/types" "node:vm" "node:zlib" "node:worker_threads" "ws" "node:v8" 
Elapsed: 169394ms | User: 6322ms | Sys: 835ms                                                                                                                                           3:33:36 PM
RSS: 0.75GB | Peak: 0.75GB | Commit: 0.84GB | Faults: 293

panic: Segmentation fault at address 0x11
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

To send a redacted crash report to Bun's team,
please file a GitHub issue using the link below:

 https://bun.report/1.1.12/Mn143f0913A6pihoH__2hgv5Buo2ldujx8NutphQ+1u8Nm/03L2okugBmpzigCA2AiB

error: script "dev" was terminated by signal SIGTRAP (Trace or breakpoint trap)
zsh: trace trap  bun --bun run dev
```

### Stack Trace (bun.report)

Bun v1.1.12 ([`43f0913`](<https://github.com/oven-sh/bun/tree/43f0913c3875cbf260c60868268adc5953b09a80>)) on macos aarch64 [RunAsNodeCommand]

Segmentation fault at address 0x00000011

- *2 unknown/js code*
- `JSC::profiledCall`
- `JSObjectCallAsFunctionReturnValue`
- `src.bun.js.api.bun.socket.Handlers.callErrorHandler`
- `src.bun.js.api.bun.socket.NewSocket`
- `src.deps.uws.NewSocketHandler`
- `TCPSocketClass__finalize`
- `WebCore::JSTCPSocket::destroy`
- `JSC::VM::~VM`

**Comment by Jarred-Sumner** [comment]
Fixed in Bun v1.1.13

**Comment by ghyath5** [comment]
> Fixed in Bun v1.1.13

when it will be released ? any expectation ?
